### PR TITLE
Add ETS Apollo parts from BDB

### PR DESCRIPTION
**Pull request by Capkirk123**
Add Apollo parts from Eyes Turned Skyward, mainly taken from BDB. Parts added include:

- Apollo Block III+ CM (Using DECQ model)

- Apollo CM Decoupler (from BDB)

- Apollo Block III+ Mission Module (from BDB)

- Apollo Block IV Mission Module (from BDB)

- Apollo Docking Probe (from BDB)

- APAS-89/95 docking port (active and passive) (from BDB)

- Apollo Block III SPS (from BDB)

- Apollo Block III HGA (from BDB)

- Apollo Block V HGA (from BDB)

- Apollo Block V Solar Panels (from BDB)

All parts have had TU shaders added to improve their look and fit in better with the DECQ parts

### CODE REVIEW

**Comment by pap1723** [comment]
Please create a file (or better yet, add to the blank readme) of the parts and their attribution to the mod / author.

**Comment by Capkirk123** [comment]
The Readme has been updated to list the parts and credit the part authors

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockIII.cfg [hunk 1]
```diff
@@ -0,0 +1,91 @@
+PART
+{
+	name = ROC-ApolloHGABlockIII
+	module = Part
+	author = CobaltWolf, capkirk
+MODEL
+{
+	model = ROCapsules/Assets/BDB/bluedog_Apollo_Block3_HGA
```
Comment (siimav) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockIII.cfg]: indent

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockV.cfg [hunk 2]
```diff
@@ -0,0 +1,90 @@
+PART
+{
+	name = ROC-ApolloHGABlockV
+	module = Part
+	author = CobaltWolf, capkirk
+MODEL
+{
+	model = ROCapsules/Assets/BDB/bluedog_Apollo_Block5_HGA
+	rotation = 0.0, -60.0, 0.0
+}
```
Comment (siimav) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockV.cfg]: that

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloMissionModule.cfg [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+PART
+{
+	name = ROC-ApolloMissionModule4
+	module = Part
+	author = CobaltWolf
+MODEL
+{
+	model = ROCapsules/Assets/BDB/bluedog_Apollo_Block4_MissionModule
+}
```
Comment (siimav) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloMissionModule.cfg]: and so on

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloActiveDockingMechanism.cfg [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	bulkheadProfiles = size0
+
+	tags = berth capture connect couple dock fasten join moor socket Apollo Kane block ?2 ?3 ?4 ?5 ?ii ?iii ?iv ?v aardv skylab hokulani
+
+	MODULE
+	{
+		name = ModuleDockingNode
+		nodeType = Apollo
+		gendered = True
+        genderFemale = False
```
Comment (siimav) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloActiveDockingMechanism.cfg]: fix

**Pull request merged**